tests/integrations/celery/test_celery.py

```diff
@@ -209,7 +209,17 @@ def dummy_task(x, y):
     else:
         assert execution_event["contexts"]["trace"]["status"] == "ok"
 
-    assert execution_event["spans"] == []
+    assert len(execution_event["spans"]) == 1
+    assert (
```

Comment:
This assert uses a `>=` on the dictionary items, which asserts that the `execution_event["spans"][0].items()` is a superset of the items of the dictionary provided in the test. So, this assert checks that all of the key-value pairs I listed (`trace_id`, `same_process_as_parent`, `op`, `description`, and `data`) are present in the `execution_event["spans"][0]` dictionary, and that each of them have the values I specified. However, the `execution_event["spans"][0]` can contain any additional key-value pairs.

The reason I wrote the assert this way was to be explicit about checking only what is relevant to this test case. The assert that was already here (on line 223) solves this problem by looking up the items we don't care about (e.g. `parent_span_id`) in the dictionary itself, which I find confusing. My way should also be more maintainable, since no changes to the test are required if we ever add additional keys to the span dictionary.

Reply:
Now I got it! Thanks for explaining.